PlanningPokerBackend/Controllers/PlayTablesController.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PlanningPokerBackend.Models;
+using PlanningPokerBackend.Models.PostRequestBodyModels;
+
+namespace PlanningPokerBackend.Controllers
+{
+    [Route("api/[Controller]/[Action]")]
```

Comment:
this should be the default route, are you sure this line is needed here?

Reply:
Routes should be configured in Starup class, but I didn't care about it yet ;)